files/build_templates/docker_image_ctl.j2

```diff
@@ -76,40 +76,40 @@ function postStartAction()
         link_namespace $DEV
     fi
     # Wait until redis starts
-    /usr/bin/docker exec database ping_pong_db_insts
+    /usr/bin/docker exec database$DEV ping_pong_db_insts
     if [[ ("$BOOT_TYPE" == "warm" || "$BOOT_TYPE" == "fastfast") && -f $WARM_DIR/dump.rdb ]]; then
         rm -f $WARM_DIR/dump.rdb
     else
         # If there is a config_db.json dump file, load it.
-        if [ -r /etc/sonic/config_db.json ]; then
-            if [ -r /etc/sonic/init_cfg.json ]; then
-                sonic-cfggen -j /etc/sonic/init_cfg.json -j /etc/sonic/config_db.json --write-to-db
+        if [ -r /etc/sonic/config_db$DEV.json ]; then
+            if [ -r /etc/sonic/init_cfg$DEV.json ]; then
```

Comment:
Do we intend to have multiple init_cfg.json files (init_cfg$DEV.json)? Or will the same init_cfg.json file be used for all instances?

Reply:
We will use same init_cfg.json, updated to code to reflect this change.